plugins/dbviewer/frontend/public/javascripts/countly.views.js

```diff
@@ -212,9 +214,13 @@
                     if (force) {
                         this.isLoading = true;
                     }
-                    this.tableStore.dispatch("fetchDbviewerTable", {_silent: !force}).then(function() {
-                        self.isLoading = false;
-                    });
+                    if (force || !this.isFetching) {
```

Comment:
I did not change this line (isFetching flag logic) due to it is included in [next branch](https://github.com/Countly/countly-server/blob/next/plugins/dbviewer/frontend/public/javascripts/countly.views.js#L222)
Also without this, it is giving same error recursively